### ISSUE TITLE
'Part 5 - Inference and Validation (Exercises).ipynb' improvements

### ISSUE THREAD

**Issue by HaleTom** [udacity/deep-learning-v2-pytorch#76]
## 'Part 5 - Inference and Validation (Exercises).ipynb'

Replace:

```
accuracy = torch.mean(equals.type(torch.FloatTensor))
```

with:

```
accuracy = torch.mean(equals.float())
```

---------------------

For the test set accuracy, why bother with the exponentiation:
```
ps = torch.exp(log_ps)
top_p, top_class = ps.topk(1, dim=1)
```

Why not just:

```
top_p, top_class = log_ps.topk(1, dim=1)
```

**Comment by GarrisonD** [comment]
Or even:

```python
accuracy = equals.float().mean()
```

**Comment by jchernus** [comment]
Happy to approve a pull request with @GarrisonD's suggestion.

**Comment by andrewng88** [comment]
Side track..

for

`equals = top_class == labels`

should it be

`equals = top_class == labels.view(-1,64)`

rather than

`equals = top_class == labels.view(*top_class.shape)`

because the former has a shape of (64,64) based on the explanation of

> equals will have shape (64, 64), try it yourself. What it's doing is comparing the one element in each row of top_class with each element in labels which returns 64 True/False boolean values for each row.

This is because of broadcasting.

Cheers.
